docs/ReleaseNotes.md

```diff
@@ -1,50 +1,93 @@
 # IPDK Kubernetes* Infrastructure Offload Release Notes
 
-## What's New in This Release
+## Releases
 
-### IPDK 23.07 K8s-Infra-Offload Component
+### IPDK 23.07
 
-- This is the release of K8s-Infra-Offload recipe for ES2K and DPDK targets.
-This is the first release that supports both these targets.
+- This is the first release of K8s-Infra-Offload recipe that supports ES2K and DPDK targets.
 
 ### Highlights
 
-#### DPDK and ES2K targets
-- Support for Kubernetes Container Network Interface (CNI) to enable pods to
-  send/receive traffic.
-- Intra Node L3 Forwarding to enable pod to pod communication, on the same node,
-  via CNI interfaces.
+#### ES2K target
+
+- Support for Kubernetes Container Network Interface (CNI) to deploy pods and
+  enable pod-to-pod connectivity on a P4 target using hardware device interfaces.
+- Use of internal gateway with dummy MAC to enable layer-3 connectivity on the same node.
+- Support for SRIOV and dynamic Subfunctions on ES2K.
```

Comment:
Is SRIOV working and tested?

Reply:
It seems to be working but SQA hasn't qualified. So, bugs can be raised.